rocketpy/rocket/aero_surface.py

```diff
@@ -34,6 +34,30 @@ def __init__(self, name):
         self.name = name
         return None
 
+    # Defines beta parameter
+    def _beta(_, mach):
+        """Defines a parameter that is often used in aerodynamic
+        equations. It is commonly used in the Prandtl factor which
+        corrects subsonic force coefficients for compressible flow.
+
+        Parameters
+        ----------
+        mach : int, float
+            Number of mach.
+
+        Returns
+        -------
+        beta : int, float
+            Value that characterizes flow speed based on the mach number.
+        """
+
+        if mach < 0.8:
+            return np.sqrt(1 - mach**2)
+        elif mach < 1.1:
+            return np.sqrt(1 - 0.8**2)
+        else:
+            return np.sqrt(mach**2 - 1)
+
```

Comment:
Where did you get this from? Probably other software, right?

Should we explicit our references?
If you took it from other software, might be a good idea to search for their references as well.